.buildkite/nightly-benchmarks/scripts/run-performance-benchmarks.sh

```diff
@@ -390,35 +400,39 @@ run_serving_tests() {
         echo "now qps is $qps"
       fi
 
-      new_test_name=$test_name"_qps_"$qps
-
-      # pass the tensor parallel size to the client so that it can be displayed
-      # on the benchmark dashboard
-      client_command="vllm bench serve \
-        --save-result \
-        --result-dir $RESULTS_FOLDER \
-        --result-filename ${new_test_name}.json \
-        --request-rate $qps \
-        --metadata "tensor_parallel_size=$tp" \
-        $client_args $client_remote_args "
-
-      echo "Running test case $test_name with qps $qps"
-      echo "Client command: $client_command"
-
-      bash -c "$client_command"
-
-      # record the benchmarking commands
-      jq_output=$(jq -n \
-        --arg server "$server_command" \
-        --arg client "$client_command" \
-        --arg gpu "$gpu_type" \
-        '{
-          server_command: $server,
-          client_command: $client,
-          gpu_type: $gpu
-        }')
-      echo "$jq_output" >"$RESULTS_FOLDER/${new_test_name}.commands"
-
+      # iterate over different max_concurrency
+      for max_concurrency in $max_concurrency_list; do
```

Comment:
Hmmm, similar concern as PP, GPU tests don't set ```max_concurrency_list```. Perhaps we should set ```max_concurrency_list=[num_prompts]``` when it is not provided.

Reply:
make senses. changed them accordingly.